src/goto-programs/instrument_preconditions.h

```diff
@@ -12,9 +12,17 @@ Date:   September 2017
 #ifndef CPROVER_GOTO_PROGRAMS_INSTRUMENT_PRECONDITIONS_H
 #define CPROVER_GOTO_PROGRAMS_INSTRUMENT_PRECONDITIONS_H
 
+#include <util/replace_symbol.h>
+
 class goto_functiont;
 class goto_modelt;
 
+replace_symbolt actuals_replace_map(
+  const exprt &,
+  const exprt &,
+  const exprt::operandst &,
+  const namespacet &);
```

Comment:
Where is this function called in this PR? I did a quick search but couldn't find ...

Reply:
This was debugging leftovers. Sorry. Removed it.